index.js

```diff
@@ -135,10 +135,10 @@ module.exports = function() {
     // Test number
     out.push('  total:     ' + res.asserts.length + '\n');
     // Pass number
-    out.push(format.green('  passing:   ' + res.pass.length + '\n'));
+    out.push(format.green('  passing:   ' + res.pass.length) + '\n');
     // Fail number
     if (res.fail.length > 0) {
-      out.push(format.red('  failing:   ' + res.fail.length + '\n'));
+      out.push(format.red('  failing:   ' + res.fail.length) + '\n');
```

Comment:
@scottcorgan Just in case you didn't notice, I made some small changes here. The problem with formatting the text, including `\n`, is that the color code appears after `\n`. Then, when splitting the output into lines to remove the `duration ...` line, the color code is removed with that line. 
This change doesn't affect the ouput much but make e2e test easier.